src/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -126,6 +126,9 @@ public static partial class Queryable
         public static TSource SingleOrDefault<TSource>(this System.Linq.IQueryable<TSource> source) { throw null; }
         public static TSource SingleOrDefault<TSource>(this System.Linq.IQueryable<TSource> source, System.Linq.Expressions.Expression<System.Func<TSource, bool>> predicate) { throw null; }
         public static System.Linq.IQueryable<TSource> Skip<TSource>(this System.Linq.IQueryable<TSource> source, int count) { throw null; }
+#if netcoreapp
```

Comment:
@weshaggard apologies if this was already pointed out in the discussion of the changes and I missed it. Does this mean we can leave out `#if` blocks on all API additions, or is there something specific to some projects?